platform/ios/src/MGLMapView.h

```diff
@@ -1118,8 +1118,10 @@ MGL_EXPORT IB_DESIGNABLE
  Converts a rectangle in the given view’s coordinate system to a geographic
  bounding box.
  
- If a longitude is less than −180 degrees or greater than 180 degrees, the
- bounding box straddles the antimeridian or international date line.
+ To bring both sides of the antimeridian or international date line into view,
+ specify some longitudes less than −180 degrees or greater than 180 degrees. For
+ example, to show both Tokyo and San Francisco simultaneously, you could set the
+ visible bounds to extend from (35.68476, −220.24257) to (37.78428, −122.41310).
```

Comment:
I think you meant to use this note in the next method but keep the existing note for this method. One version is for when the return value is an MGLCoordinateBounds; the other is for when the first parameter is an MGLCoordinateBounds.